stock_by_warehouse/static/src/components/warehouse/warehouse_field.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+
+    <t t-name="stock_by_warehouse.ShowWarehouseInfo" owl="1">
+        <div>
+            <p t-if="this.props.byLocation" class="availability_product my-0 mr-2">
+                The product is available in: <t t-out="show" /> locations.
+                <a
+                    role="button"
+                    tabindex="0"
+                    class="js_available_info fa fa-bars"
+                    t-on-click.stop="(ev) => this.onClick(ev)"
+                />
+            </p>
+            <a
+                t-else=""
+                role="button"
+                tabindex="0"
+                class="js_product_warehouse"
+                t-on-click.stop="(ev) => this.onClick(ev)"
+            >
+                <b t-out="show" /> ... see all warehouses .. <i class="fa fa-list-ul" />
+            </a>
+        </div>
+    </t>
+
+    <t t-name="stock_by_warehouse.ProductWarehousePopOver" owl="1">
+        <h3 t-if="this.props.title" class="o_popover_header" t-out="this.props.title" />
+        <div class="popover-body stock-by-warehouse-widget">
+            <div class="panel clearfix container">
+                <div class="list-group">
+                    <t t-foreach="this.props.lines" t-as="line" t-key="line_index">
+                        <div
+                            t-if="
+                                line.available_not_res
+                                &gt; 0 or line.available
```

Comment:
One condition per line, please, currently conditions are split, half on on a line and the other alf in the next one